### ISSUE TITLE
Steps with no `uses` clause aren't recognized

### ISSUE THREAD

**Issue by ForNeVeR** [YunaBraska/github-workflow-plugin#25]
**What happened?**
There's a problem with recognition of step ids that aren't declaring any `using` clauses.

**How can we reproduce the issue?**
For example, see this config:
```yml
name: Release
on:
  push:
    tags:
      - '*'

jobs:
  docker-push:
    runs-on: 'ubuntu-20.04'
    steps:
      - name: Read version from ref
        id: version
        shell: pwsh
        run: echo "version=$(./Scripts/Get-Version.ps1 -RefName $env:GITHUB_REF)" >> $env:GITHUB_OUTPUT

      - name: Build and push Docker image
        uses: docker/build-push-action@v4
        with:
          push: true
          tags: test:v${{ steps.version.outputs.version }}
```

It will say that `version` is undefined, while it is defined by a previous step, `id: version`
<img width="307" alt="image" src="https://github.com/YunaBraska/github-workflow-plugin/assets/92793/a23aec99-f8e0-4be2-b9c7-bc2913e7e67d">

(I do understand that it's impossible to derive what **outputs** such a step would have, though.)

**Relevant log output**
> N/A

**Operating System**
> Windows 11

**Project Version**
> 2.2.7

**Expected behavior**
The plugin should properly recognize the steps with no `uses` clause

**Comment by YunaBraska** [comment]
I am currently doing a big refactoring for stabilisation and better maintainability. 
Therefore, I need to ask you for some patience. 🙏  
I guess not detected step IDs will be fixed within the refactoring #18. 

> I do understand that it's impossible to derive what outputs such a step would have, though.

That's a very good example of an output variable which I also don't match. It's hard, but not impossible, i will take this example with me. 👍

**Comment by YunaBraska** [comment]
Thx for reporting. It should be fixed now[3.0.0](https://plugins.jetbrains.com/plugin/21396-github-workflow/versions/stable/403332) Feel free to open new tickets on any bugs or feature requests.
